tests/e2e/tidbcluster/serial.go

```diff
@@ -107,7 +107,7 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 	})
 
 	// tidb-operator with AdvancedStatefulSet feature enabled
-	ginkgo.Context("[Feature: AdvancedStatefulSet]", func() {
+	ginkgo.Context("[Feature: AdvancedStatefulSet And Webhook]", func() {
```

Comment:
```suggestion
	ginkgo.Context("[Feature: AdvancedStatefulSet][Feature: Webhook]", func() {
```

then we can match each feature via following regexs: --ginkgo.focus='\[Feature:\sAdvancedStatefulSet\], --ginkgo.focus='\[Feature:\sWebhook\]